src/main/java/ch/naviqore/raptor/router/FootpathRelaxer.java

```diff
@@ -31,18 +31,18 @@ class FootpathRelaxer {
 
     /**
      * @param raptorRouter the current raptor instance for access to the data structures.
-     * @param objective    the best time per stop and label per stop and round.
+     * @param query        the best time per stop and label per stop and round.
```

Comment:
The param description for query should be ~ `object containing query configuration and intermediate working variables` or something of this sort.